src/TwigRenderer.php

```diff
@@ -34,7 +34,14 @@ function __construct(array $config) {
     if (isset($this->config['src']['namespaces'])) {
       foreach ($this->config['src']['namespaces'] as $namespace) {
         foreach ($namespace['paths'] as $path) {
-          $this->loader->addPath($path, $namespace['id']);
+          // The twigRenderer tries to access paths which 
+          // might be not present anymore. In that case we 
+          // have to catch this
+          try {
+            $this->loader->addPath($path, $namespace['id']);
+          } catch (Throwable $e) {
+            // ignore
```

Comment:
empty catch blocks always have be go :thinking: 

Could we report this back as some sort of notice or warning so a developer can clean up the problematic code?